as-controller-board-status/functions/output_test.go

```diff
@@ -163,20 +173,16 @@ func prepCheckControllerBoardStatusTest() (testTable []testTableCheckControllerB
 	configBadNotificationHostThresholdsExceeded.VendingEndpoint = testServerStatusOK.URL
 	configBadNotificationHostThresholdsExceeded.MaxTemperatureThreshold = temp49
 
-	// Set bad MQTT and Vending endpoints to produce specific error conditions
-	// in processTemperature, which first sends a request to MQTT, then
-	// another request to the vending endpoint
-	edgexcontextBadDoorStatusCommandEndpoint := pkg.NewAppFuncContextForTest(correlationID, lc)
-	configBadDoorStatusCommandEndpoint := getCommonApplicationSettingsTyped()
-	configBadDoorStatusCommandEndpoint.DoorStatusCommandEndpoint = testServerThrowError.URL
-	configBadDoorStatusCommandEndpoint.VendingEndpoint = testServerStatusOK.URL
-
-	// As described above, in order to produce the error condition for
-	// processTemperature failing to hit the VendingEndpoint, we have to hit
-	// the DoorStatusCommandEndpoint successfully first
+	// Set Device Command to throw error to produce specific error conditions
+	// in processTemperature, which sends a request to the vending endpoint
+	edgexcontextBadDoorStatusCommand := pkg.NewAppFuncContextForTest(correlationID, lc)
+	configBadDoorStatusCommand := getCommonApplicationSettingsTyped()
+	configBadDoorStatusCommand.VendingEndpoint = testServerStatusOK.URL
```

Comment:
This code has nothing to do with commands. Please restore original comment.

Reply:
The original comment does not make more sense - https://github.com/intel-retail/automated-vending/blob/Edgex-3.0/as-controller-board-status/functions/output_test.go#L166? There is no MQTT endpoint which is replaced by Command. These are used here - https://github.com/intel-retail/automated-vending/blob/46e3189bbf39441330d268f034b25fc115588b6e/as-controller-board-status/functions/output_test.go#L341.

Reply:
OK, missed that you also removed
```
configBadDoorStatusCommandEndpoint.DoorStatusCommandEndpoint = testServerThrowError.URL
```
Also this block is so far from the test case that uses mockErrCommandClient the comment is out of context. 
```suggestion
	// Setup for Device Command (mockErrCommandClient set below) to throw error to produce specific error conditions
	// in processTemperature, which sends a request to the vending endpoint
	edgexcontextBadDoorStatusCommand := pkg.NewAppFuncContextForTest(correlationID, lc)
	configBadDoorStatusCommand := getCommonApplicationSettingsTyped()
	configBadDoorStatusCommand.VendingEndpoint = testServerStatusOK.URL
```